app/homepage.py

```diff
@@ -48,6 +48,10 @@
 # ----------------------------------------------------------------------------------------------------------------------
 # CONTENT
 
+# Pyronear - Horizontal Logo
+pyro_logo = 'https://pyronear.org/img/logo_letters.png'
```

Comment:
Shouldn't we put this in the `assets` folder rather ? I think that it doesn't fit well here

Reply:
The idea was to have the less number of elements within the repo. As this logo is also used and hosted for the website, let's not re-upload it.